apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -265,7 +270,9 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): J
             const fieldName = t("userPortal:components.profile.fields." + schema.name.replace(".", "_"),
                 { defaultValue: schema.displayName }
             );
-
+            if (isMobileVerificationEnabled && checkSchemaType(schema.name, "mobile")) {
```

Comment:
```suggestion

            if (isMobileVerificationEnabled && checkSchemaType(schema.name, "mobile")) {
```

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1275/commits/375a1a7a8b040739aaae46cb9b9657e461a0bf70